Android/app/src/main/java/com/example/samplestickerapp/StickerPackValidator.java

```diff
@@ -26,15 +26,17 @@
 import java.util.List;
 
 class StickerPackValidator {
-    private static final int STICKER_FILE_SIZE_LIMIT_KB = 100;
+    private static final int STATIC_STICKER_FILE_LIMIT_KB = 1000;
+    private static final int ANIMATED_STICKER_FILE_LIMIT_KB = 3000;
```

Comment:
Is this the right file size limit? It is 100kb and 500 kb according to the spec?